src/screens/ChallengeDetailScreen/components/LastMonthRankingModal.tsx

```
@@ -0,0 +1,152 @@
+import React, {useEffect, useState} from 'react';
+import {Image, Modal, ModalProps, ActivityIndicator, TouchableOpacity} from 'react-native';
```

ESLint

[error] Replace `Image,·Modal,·ModalProps,·ActivityIndicator,·TouchableOpacity` with `⏎··Image,⏎··Modal,⏎··ModalProps,⏎··ActivityIndicator,⏎··TouchableOpacity,⏎`

(prettier/prettier)